bin/devnet/src/cli.rs

```diff
@@ -18,6 +18,15 @@ pub struct Cli {
     /// this will change in the future but is good enough for testing currently
     #[clap(short, long, default_value = "2")]
     pub nodes_in_network:        u64,
+    /// the secret key/address to use as the controller
+    #[clap(short, long, default_value = "7")]
+    pub anvil_key:               u16,
+    /// starting block to fork
+    #[clap(short = 's', long)]
+    pub fork_block:              Option<u64>,
+    /// fork url
+    #[clap(long, requires = "fork_block")]
+    pub fork_url:                Option<String>,
```

Comment:
can  we just set the default to whatever our reth is. Makes it 100 times easier on startup

Reply:
we don't necessarily need a fork url for the devnet, but otherwise i agree